src/components/RewardCard.jsx

```diff
@@ -136,6 +136,7 @@ const RewardCard = ({ name, desc, company, image, points, requiredPoints }) => {
             ) : (
               <Text>Reach {requiredPoints - points} pts</Text>
             )}
+            {type && <Text>Type: {type}</Text>}
```

Comment:
can u do if points exist, display points bar, if type is raffle, say "Raffle Prize" and else, display "Type: [type}"

Reply:
done, also made it so if type doesn't exist, then the `Type: [type]` doesn't show up